stormpath/stormpath-express
# LinkedIn support

LinkedIn OAuth provider support.

##### How to verify

1. Add a LinkedIn directory on your Stormpath account.
2. Start up your application and head to the login view.
3. There should be a LinkedIn link at the right of the login view.
4. Click the LinkedIn link.
5. You are now redirected to LinkedIn's login portal. Once you've successfully authenticated you should be redirected back to the URL that you were previously on.
